src/sentry/static/sentry/app/components/tagDistributionMeter.tsx

```diff
@@ -66,10 +69,18 @@ export default class TagDistributionMeter extends React.Component<Props> {
     renderLoading: () => null,
     renderEmpty: () => <p>{t('No recent data.')}</p>,
     renderError: () => null,
+    showReleasePackage: false,
```

Comment:
Should this be set to true for issue details as well?

Reply:
We don't need to show the package name there as the issue belongs to one specific project.

Reply:
We use package names to differentiate the release versions where it's not obvious to which project they belong - for example in dashboards, the 'Events by Release' widget shows also the packages because otherwise, it would just show multiple `1.0.0`s from multiple different projects.